eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -386,7 +419,7 @@ function GetExistingTags($apiUrl) {
 
     # Return an empty list if there are no tags in the repo
     if ($statusCode -eq 404) {
-      return @()
+      return ,@()
```

Comment:
What prompted this?

Reply:
Functions that return empty lists have the output coerced to `$null`, this small modification actually returns an empty list. 

This helps address a bug a couple lines down where the output is expected to be an array and we're checking whether the list contains a value. (https://github.com/Azure/azure-sdk-for-cpp/blob/c044ce0f15d692a0714270713c1a1f7b1a6a19c0/eng/common/scripts/artifact-metadata-parsing.ps1#L510) 

This approach was discovered here: https://stackoverflow.com/questions/18476634/powershell-doesnt-return-an-empty-array-as-an-array

Reply:
Stupid PS!

Does -contains fail if $existingTags is null? I know PS lets a lot of those things no-op.